source/monitoring-and-logging.txt

```diff
@@ -0,0 +1,237 @@
+.. _kafka-monitoring-logging:
+
+======================
+Monitoring and Logging
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+
+
+Sink Connector JMX Metrics
+--------------------------
+
+Use the metrics in the following table to monitor the behavior of your sink
+connector through {+jmx-long+} (JMX):
+
+.. tip:: JMX Attributes
+
+   {+jmx-hover+} represents an individual metric as an attribute of an ``MBean``.
+   To learn more about attributes and ``MBeans``, see the
+   `Standard MBeans Tutorial <https://docs.oracle.com/javase/tutorial/jmx/mbeans/standard.html>`__
+   from Oracle.
```

Comment:
Just a note for future work -- this description probably would belong in a section that introduces monitoring so it doesn't need to be duplicated for the Source Connector JMX Metrics section.